Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/roadmap.mdx

title: zkApps Roadmap
hide_title: true
sidebar_label: Roadmap

> description: Explore the zkApps roadmap detailing available features, upcoming enhancements, and future developments. This page provides insights on Mina zkApp CLI, SnarkyJS, data availability, and more.

Reviewer: ```suggestion
description: zkApps roadmap, zkApp features, enhancements, and future development plans. Insights on Mina zkApp CLI, SnarkyJS, data availability, and more.
```
